cmd/up.go

```diff
@@ -140,6 +135,14 @@ func ClusterUp(ctx context.Context, dialersOptions hosts.DialersOptions, flags c
 		}
 		kubeCluster.NewHosts = newNodes
 		reconcileCluster = true
+
+		kubeCluster.RemoveHostsLabeledToIgnoreUpgrade(ctx)
+		maxUnavailable, err := kubeCluster.ValidateHostCountForUpgrade()
```

Comment:
With `ValidateHostCountForUpgrade` returning an int, this function name should be different. Something along the line of `CalculateHostCountForUpgrade`